features/matchers/developer_uses_trigger_matcher.feature

```diff
@@ -0,0 +1,163 @@
+Feature: Developer uses trigger matcher
+  As a Developer
+  I want a trigger matcher
+  In order to validate triggered exceptions against my expectations
+
+  Scenario: "Trigger" alias matches using the trigger matcher with explicit method name
```

Comment:
Can you please rephrase the titles to describe more what this lets the developer do?

Example here: "Checking a deprecated error is triggered"